index.html

```diff
@@ -175,10 +175,12 @@ <h2 class="ui dividing header">Links</h2>
           <li class="es6"><a href="https://www.elastic.co/guide/en/elasticsearch/reference/6.x/index.html">Elasticsearch Reference</a>, official documentation;</li>
           <li class="es7"><a href="https://www.elastic.co/guide/en/elasticsearch/reference/7.x/index.html">Elasticsearch Reference</a>, official documentation;</li>
 
-          <li><a class="es5 es6 es7" href="https://www.docker.elastic.co/">Docker repository</a>, quick start any Elastic software;</li>
+          <li class="es5 es6 es7"><a href="https://www.docker.elastic.co/">Docker repository</a>, quick start any Elastic software;</li>
           <li><a href="https://dev.to/t/elasticsearch">Dev.to Elasticsearch</a>, great source of content about Elastic;</li>
           <li><a href="https://discuss.elastic.co/">Official forum</a> and <a href="http://stackoverflow.com/questions/tagged/elasticsearch">StackOverflow</a> for support;</li>
           <li class="es5"><a href="https://www.elastic.co/products/upgrade_guide">Official online migration tool</a> to help upgrading the stack to 6.x.</li>
+	  <li class="es5 es6 es7"><a href="https://github.com/moshe/elasticsearch-comrade">Elasticsearch Comrade</a>, an open-source Elasticsearch admin and monitoring panel;</li>
+	  <li class="es2 es5 es6 es7"><a href="https://github.com/lmenezes/cerebro">Cerebro</a>, an open source(MIT License) elasticsearch web admin;</li>
```

Comment:
May I suggest another wording:

Comrade: a Python based Elasticsearch web admin and monitoring panel
Cerebro: a Java based Elasticsearch web admin